source/MRMesh/MRBitSet.h

```diff
@@ -25,6 +25,8 @@ class BitSet : public boost::dynamic_bitset<std::uint64_t>
     MRMESH_API BitSet & operator ^= ( const BitSet & b );
     MRMESH_API BitSet & operator -= ( const BitSet & b );
 
+    MRMESH_API IndexType find_last() const;
```

Comment:
please add a comment, what it returns both when there are 1 bits, and when all bits are zero

Reply:
fixed